### PR TITLE
Merge Accord trims

### PR DESCRIPTION
**Pull request by sshane**
Similar cars with different transmissions

### CODE REVIEW

File: selfdrive/car/honda/values.py [hunk 1]
```diff
@@ -35,7 +35,6 @@ class CruiseButtons:
 
 class CAR:
   ACCORD = "HONDA ACCORD 2T 2018"
```
Comment (ShaneSmiskol) [inline selfdrive/car/honda/values.py:37]: Do we want to remove the 2T in the name to generalize it?

Comment (adeebshihadeh) [reply selfdrive/car/honda/values.py:37]: yes

**Comment by ShaneSmiskol** [comment]
@pd0wm The two Accords use GEARBOX messages on different addresses. I checked to see if one is also available on the other and sadly, their exclusive to their cars. Also checked `392 GEARBOX:` and it wasn't available either

**Comment by pd0wm** [comment]
As discussed on slack, if we have to we should do something similar to VW: https://github.com/commaai/openpilot/blob/master/selfdrive/car/volkswagen/interface.py#L35

File: selfdrive/car/honda/interface.py [hunk 2]
```diff
@@ -138,6 +139,10 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[]):  # py
     if candidate == CAR.CRV_5G:
       ret.enableBsm = 0x12f8bfa7 in fingerprint[0]
 
+    # Accord 1.5T CVT has different gearbox message
+    if candidate == CAR.ACCORD and 0x191 in fingerprint[0]:
+      ret.transmissionType = TransmissionType.direct  # FIXME: no cvt trans type
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/interface.py:144]: add this to cereal

File: selfdrive/car/honda/values.py [hunk 3]
```diff
@@ -164,25 +173,43 @@ class CAR:
...
+      b'57114-TVE-H250\x00\x00',
     ],
     (Ecu.eps, 0x18da30f1, None): [
-      b'39990-TVA,A150\x00\x00', # modified firmware
-      b'39990-TVA-A150\x00\x00',
-      b'39990-TVA-A160\x00\x00',
+      b'39990-TVA,A150\x00\x00',  # modified firmware
+      b'39990-TVA-A150\x00\x00',  # Are these two different steerRatio?
+      b'39990-TVA-A160\x00\x00',  # Sport, Sport 2.0T and Touring 2.0T have different ratios
       b'39990-TVA-A340\x00\x00',
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/values.py:208]: don't leave these comments in

File: selfdrive/car/honda/carstate.py [hunk 4]
```diff
@@ -196,7 +199,11 @@ class CarState(CarStateBase):
   def __init__(self, CP):
     super().__init__(CP)
     can_define = CANDefine(DBC[CP.carFingerprint]["pt"])
-    self.shifter_values = can_define.dv["GEARBOX"]["GEAR_SHIFTER"]
+    self.gearbox_name = "GEARBOX"
```
Comment (adeebshihadeh) [inline selfdrive/car/honda/carstate.py:202]:
```suggestion
    self.gearbox_msg = "GEARBOX"
```

File: selfdrive/test/test_routes.py [hunk 5]
```diff
@@ -49,7 +49,7 @@
   TestRoute("03be5f2fd5c508d1|2020-04-19--18-44-15", HONDA.HRV),
   TestRoute("917b074700869333|2021-05-24--20-40-20", HONDA.ACURA_ILX),
   TestRoute("81722949a62ea724|2019-04-06--15-19-25", HONDA.ODYSSEY_CHN),
-  TestRoute("08a3deb07573f157|2020-03-06--16-11-19", HONDA.ACCORD_15),
+  TestRoute("08a3deb07573f157|2020-03-06--16-11-19", HONDA.ACCORD),  # 1.5T
```
Comment (adeebshihadeh) [inline selfdrive/test/test_routes.py:52]:
```suggestion
  TestRoute("08a3deb07573f157|2020-03-06--16-11-19", HONDA.ACCORD),
```

File: selfdrive/test/test_routes.py [hunk 6]
```diff
@@ -49,7 +49,7 @@
...
   TestRoute("1da5847ac2488106|2021-05-24--19-31-50", HONDA.ACCORD),
```
Comment (pd0wm) [inline selfdrive/test/test_routes.py:53]:
```suggestion
  TestRoute("1da5847ac2488106|2021-05-24--19-31-50", HONDA.ACCORD),  # 2.0T
```

**Pull request merged**